starfish/ddo/authentication.py

```diff
@@ -0,0 +1,91 @@
+"""
+    Authentication Class
+    To handle embedded public keys
+"""
+
+import json
+import re
+
+from squid_py.ddo.public_key_base import PublicKeyBase
+
+
+class Authentication:
```

Comment:
Is this class meant to hold auth info for Surfer as well?

Reply:
At the moment only for DDO authentication of a DDO record